hudi-common/src/main/java/org/apache/hudi/avro/HoodieAvroUtils.java

```diff
@@ -454,6 +457,10 @@ public static String getNestedFieldValAsString(GenericRecord record, String fiel
    * Obtain value of the provided field, denoted by dot notation. e.g: a.b.c
    */
   public static Object getNestedFieldVal(GenericRecord record, String fieldName, boolean returnNullIfNotFound) {
+    if (StringUtils.isNullOrEmpty(fieldName.trim())) {
```

Comment:
why here the fieldName be empty if we use default field

Reply:
> 

For the sake of logical robustness, to prevent users from calling this method function, passing in vacancy will cause an error.